scss/tech.scss

```diff
@@ -20,16 +20,22 @@
     width: 100%;
 
     &::before {
-      content: '';
       border: 1px solid $training-green;
       border-radius: 5px;
+      bottom: 10px;
+      content: '';
       height: 100%;
-      right: 10px;
       position: absolute;
-      bottom: 10px;
+      right: 10px;
+      transition: all 500s ease;
```

Comment:
500s? that transition would be really slow, I think what you want is `0.5s`, I'd also use top and left as properties instead of bottom right to transition those instead of `all` by specifying the properties the transition will be more performant